### PR TITLE
Fix GIF Disappear when return from background

### PR DESCRIPTION
**Pull request by ywz2010**
GIF Image will disappear after press the home button and return back.  
Set `removedOnCompletion` to be`false` will fix  all `CAAnimation` disappear like stopping after going into the background.

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @nicklockwood to be a potential reviewer.

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @nicklockwood as a potential reviewer. Could you take a look please or cc someone with more context?

File: Libraries/Image/RCTGIFImageDecoder.m [hunk 1]
```diff
@@ -91,6 +91,7 @@ - (RCTImageLoaderCancellationBlock)decodeImageData:(NSData *)imageData
     animation.keyTimes = keyTimes;
     animation.values = images;
     animation.duration = duration;
+    animation.removedOnCompletion = false;
```
Comment (javache) [inline Libraries/Image/RCTGIFImageDecoder.m]: Can you make this NO instead of false?

Comment (ywz2010) [inline Libraries/Image/RCTGIFImageDecoder.m]: @javache  Hi, I have change false to NO.

**Comment by ide** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1538523759788537/int_phab) to review.

**Comment by narychen** [comment]
I encounter another GIF bug on IOS. 
If you mount a GIF Image which size is bigger than 250k, you mount it and unmount it a few times,
the gif image will not show.
And at the meantime, the other static image which size is bigger than 250k will not render either.
I tested a lot of times and found the 250k is the boundary of this problem.
On android everything goes well.

**Comment by narychen** [comment]
Anyone here could help?

**Comment by lukeforehand** [comment]
Can we reopen this issue?  It is still happening.  A workaround is to add '?' + Math.random() to the source uri.

**Pull request closed without merging**